### PR TITLE
feature/amazon-ads-v0.2.0

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
Update amazon ads package dependency version

**This PR will result in the following new package version:**

v1.5.0

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**
Update amazon ads version in the packages.yml and documentation

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test (except for the known google ads issue)
- [x] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

none required

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] ~DECISIONLOG updates have been updated (if applicable)~
- [ ] ~Appropriate yml documentation has been added (if applicable)~

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [ ] docs were regenerated (unless this PR does not include any code or yml updates)
Wait until all packages deps updates are merged into the release branch

### If you had to summarize this PR in an emoji, which would it be?
 
🚚

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -26,8 +26,11 @@ packages:
   - package: fivetran/twitter_ads
     version: [">=0.6.0", "<0.7.0"]
 
-  - package: fivetran/amazon_ads
-    version: [">=0.1.0", "<0.2.0"]
+  # - package: fivetran/amazon_ads
+  #   version: [">=0.2.0", "<0.3.0"]
+  - git: https://github.com/fivetran/dbt_amazon_ads.git
+    revision: feature/api-v3-updates
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:33]: Reminder to switch before merge.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
@fivetran-catfritz thanks for pushing these Amazon Ad updates along! This PR looks good to go!

One thing I noticed is that we are not testing Amazon Ads in our integration tests pipeline.

https://github.com/fivetran/dbt_ad_reporting/blob/050495a2c0e34fe7ef4daccba0dd6e2cad09b777/.buildkite/scripts/run_models.sh#L20-L21

I tested locally and all checked out, but I do wonder what our approach should be when making updates to excluded models from the integration tests? We intentionally exclude them to improve the integration test timeline. But I do worry in the future if we accidentally introduce an issue because we weren't testing.

What are your thoughts on manually adding the package models we are updating, but then removing a packages models that are not being updated for Ad Reporting? My only concern is that I wouldn't want to remove a package like Google Ads or Facebook Ads as they are most popular and should always be tested. 

Any thoughts?

**Comment by fivetran-catfritz** [comment]
> What are your thoughts on manually adding the package models we are updating, but then removing a packages models that are not being updated for Ad Reporting? My only concern is that I wouldn't want to remove a package like Google Ads or Facebook Ads as they are most popular and should always be tested.

I think that would make sense and be doable. Maybe this can be something we update the PR checklist specifically for ad_reporting? Something like:
- Adjust run_models dbt run and dbt test variables to include:
  - This package = true
  - Google Ads and Facebook Ads = true
  - All other packages = false
?

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-catfritz FYI buildkite is now failing due to Pinterest Ads version dep out of range. This will be resolved once the Pinterest changes are live. Please feel comfortable merging to the release branch with the failed buildkite job. We will confirm all tests pass once all upstream package updates are live on the dbt hub and before merging the release branch to `main`.

**Pull request merged**